src/use_case/settings/SettingsOutputData.java

```diff
@@ -0,0 +1,18 @@
+package use_case.settings;
+
+import entity.User;
+
+public class SettingsOutputData {
+
+  User user;
+  private final boolean useCaseFailed;
+
+  public SettingsOutputData(User user, boolean useCaseFailed) {
+    this.user = user;
+    this.useCaseFailed = useCaseFailed;
```

Comment:
Shouldn't this be a failure string/error message?

Reply:
Not for now